eng/testing/linker/trimmingTests.targets

```diff
@@ -90,6 +90,7 @@
                                                  .Replace('{AdditionalProperties}', '$(_additionalPropertiesString)')
                                                  .Replace('{RuntimeHostConfigurationOptions}', '$(_runtimeHostConfigurationOptionsString)')
                                                  .Replace('{AdditionalProjectReferences}', '$(_additionalProjectReferencesString)')
+                                                 .Replace('{DefaultNetCoreTargetFramework}', '$(DefaultNetCoreTargetFramework)')
```

Comment:
You shouldn't need this, you can just use `{TargetFramework}`, which is already a parameter to the template.

Reply:
Yeah, this was a shot in the dark when I didn't know why the update wasn't working. Will remove it now.